main.nf

```diff
@@ -16,6 +16,20 @@ if (params.help) {
     exit 1
 }
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    input handling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+// handle initial weights as optional input file(s)
+// if it is not provided, set it to an empty list
+if (params.initial_weights == null) {
```

Comment:
move this if inside the unnamed workflow definition just below. please is just for my mental problems.

Reply:
I have done it like this, because in nf-core they always put the input parametesr handling section outside on top